hw/ip/otbn/data/otbn.hjson

```diff
@@ -127,9 +127,39 @@
     { name: "BUS.INTEGRITY",
       desc: "End-to-end bus integrity scheme."
     }
+    { name: "CONTROLLER.FSM.GLOBAL_ESC",
+      desc: "The controller FSM moves to a terminal error state upon global escalation."
+    }
+    { name: "CONTROLLER.FSM.LOCAL_ESC",
+      desc: '''
+        The controller FSM moves to a terminal error state upon local escalation.
+        Can be triggered by CONTROLLER.FSM.SPARSE.
```

Comment:
I am not entirely sure anymore but if I remember correctly, the controller FSM will also escalate if the other two FSMs see an invalid state. If this is really the case, it would make sense to also list this here, i.e., `Can be triggered by CONTROLLER.FSM.SPARSE, SCRAMBLE_CTRL.FSM.SPARSE, START_STOP_CTRL.FSM.SPARSE`.

Reply:
Indeed, thanks!